docker/dev/Dockerfile

```
@@ -0,0 +1,25 @@
+# Base image
+FROM node:18-alpine
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package.json and package-lock.json
+COPY ../../package.json .
```

this would change to look like 
```suggestion
COPY ./package.json .
```

---

that's right but according to project folder structure that's how it must be followed

---

okay